files/pt-br/web/css/css_box_model/introduction_to_the_css_box_model/index.html

```diff
@@ -10,7 +10,7 @@ <h2 id="Resumo">Resumo</h2>
 
 <p>No CSS, cada um desses <em>boxes </em>retangulares é descrita usando o <em>box model</em> padrão. Este modelo descreve o conteúdo do espaço ocupado por um elemento. Cada box possui 4 edges: <strong>margin edge</strong>, <strong>border edge</strong>, <strong>padding edge</strong> e <strong>content edge</strong>.</p>
 
-<p><img alt="CSS Box model" src="https://mdn.mozillademos.org/files/8685/boxmodel-(3).png" style="height: 384px; width: 548px;"></p>
+<p><img alt="CSS Box model" src="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model/boxmodel-(3).png" style="height: 384px; width: 548px;"></p>
```

Comment:
Inicialmente obrigado pela contribuição!

Gostaria de sugerir que, ao invés de utilizar o caminho absoluto, utilizar um caminho relativo.

```suggestion
<p><img alt="CSS Box model" src="/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model/boxmodel-(3).png" style="height: 384px; width: 548px;"></p>
```

O que você acha?